.travis.yml

```diff
@@ -1,9 +1,12 @@
 language: python
 python:
   - '2.7'
-before_install:
```

Comment:
It was logging in that was the problem. PR builds only run the local tests, so we won't see it until the merge.